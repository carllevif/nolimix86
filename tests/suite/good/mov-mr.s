  .text
mov 0x0(%eax), %eax