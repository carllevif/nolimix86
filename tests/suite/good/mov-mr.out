  .text
start:
mov 0x0(%eax), %eax
